odkx-src/notify-using.rst

```diff
@@ -0,0 +1,147 @@
+Using ODK-X Notify
+=======================
+
+Skunkworks-Parrot (Desktop App):
+---------------------------------
+
+Creating Notification:
+"""""""""""""""""""""""
+
+1. Launch the Desktop application(Skunkworks-Parrot)
+2. Enter the Username and Password in the provided field and click :menuselection:`Login`.
+  
+  .. image:: /img/notify-using/notify-login.*
+
+.. note::
+
+   Only users with the default group “ROLE_SITE_ACCESS_ADMIN” are allowed to log in to the Desktop application.
+
+3. After a successful login, the user will be redirected to “Create Notification” window as shown below.
+
+  .. image:: /img/notify-using/notify-create-notification.*
+
+4. Enter Title and the message for the Notification in the provided fields.
+
+5. Users can also send an image with the notification. To send the image click the :menuselection:`Select` button under the Image tab and select the image from your PC. Sending image with the notification is optional.
+
+6. Select the Notification group to which you want to send a notification.
+
+7. Select notification type and click :menuselection:`Send`.
+
+.. note::
+
+  **Simple Notification:** Recipients of simple notification can not send a reply to that notification.
+
+  **Interactive Notification:** Recipients of interactive notification can send a reply to that notification which admin can view in Desktop app.
+
+
+Creating Group:
```

Comment:
Is this explaining the roles mentioned above? May need to move up. Are these the same/different than the permission filter ones? unclear.

Reply:
Desktop App users can create groups apart from default groups(permission filter groups). I added one example of that. If still unclear, please let me know.